components/configuration/configuration_derive/src/lib.rs

```diff
@@ -230,6 +232,43 @@ fn diff(fields: &Punctuated<Field, Comma>, creat_name: &Ident) -> Result<TokenSt
     })
 }
 
+fn typed(fields: &Punctuated<Field, Comma>, creat_name: &Ident) -> Result<TokenStream> {
```

Comment:
Could you add some comments about the `typed`?

Reply:
I had added comments on the `Configuration` trait, PTAL.